### DISCUSSION TITLE
如何解决防止QQ被冻结的问题

### DISCUSSION THREAD

**Discussion by thechex1** [lss233/chatgpt-mirai-qq-bot#128] [Q&A]
使用了半天就被冻结了

**Comment by lss233** [comment]
新号别急着用，少加群，少发消息，挂一段时间获取信任。  
也有可能是被人举报了。

**Comment by CiaLion** [comment]
我是每次开mirai都手动登录最后协议用ANDROID_WATCH
login account password ANDROID_WATCH
你可以试试

**Comment by King-Of-Knights** [comment]
@CiaLion 我试了，新号，目前是涉嫌违规，官方正在核实，还没封
